gsa/src/gmp/model.js

```diff
@@ -105,8 +104,15 @@ class Model {
       delete copy.in_use;
     }
 
-    if (isDefined(element.owner) && isEmpty(element.owner.name)) {
-      delete copy.owner;
+    if (isDefined(element.owner)) {
+      if (this.entityType === 'task') {
```

Comment:
I really don't want such if checking in the generic model class. Why is this needed? Maybe just parse the owner in the task model class before calling super or just re-parse it from the original element.